src/components/ActivityCreation/ActivityCreationSideBar.tsx

```diff
@@ -79,7 +79,7 @@ const ActivityCreationSideBar = ({
           })}
         </div>
         <Link
-          className="group flex items-center gap-2 pl-6"
+          className="group flex w-fit items-center gap-2 pl-6"
           href="#ticket-setting"
```

Comment:
```suggestion
          href="#tickets"
```

Reply:
fix it : 6b854e58eccccd4c67c3c6f573ec6b97a0395010